src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -49,7 +50,17 @@ export class AddCellAction extends Action {
 	public run(context: INotebookEditor): Promise<boolean> {
 		return new Promise<boolean>((resolve, reject) => {
 			try {
-				context.addCell(this.cellType);
+				//Add Cell after current selected cell.
+				let notebookcomponent = context as NotebookComponent;
+				let id = notebookcomponent.activeCellId;
+				let index = undefined;
```

Comment:
If context or context.cells or id is null/undefined, then index will still be undefined. may want to set it to a real value initially.

Reply:
Done